src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTabOrFirstId = tabs.find(tab => tab.id === activeTabId)
+    ? activeTabId
+    : tabs[0].id;
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            return (
+              <li
+                key={tab.id}
+                data-cy="Tab"
+                className={`${activeTabOrFirstId === tab.id ? 'is-active' : ''}`}
```

Comment:
```suggestion
                className={activeTabOrFirstId === tab.id ? 'is-active' : ''}
```

we can write without these backtick